rubix-hadoop2/pom.xml

```diff
@@ -17,10 +17,17 @@
 
         <dependency>
             <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-core</artifactId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
Can you provide some comment why we have to change this

Reply:
`hadoop-core` is a hadoop1 dependency, whereas `hadoop-common` is the hadoop2 version of this dependency. Steps such as initializing the Yarn configuration for Hadoop2ClusterManager requires the dependency upgrade.

Reply:
ok. sounds good